### PR TITLE
Add test for JobEventRdbSearch

### LINKED ISSUE #1254: Add test for JobEventRdbSearch

### PR DESCRIPTION
**Pull request by Technoboy-**
Fixes #1254.

### CODE REVIEW

File: elasticjob-cloud/elasticjob-cloud-scheduler/src/test/java/org/apache/shardingsphere/elasticjob/cloud/console/controller/search/JobEventRdbSearchTest.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class JobEventRdbSearchTest {
```
Comment (dongzl) [inline elasticjob-cloud/elasticjob-cloud-scheduler/src/test/java/org/apache/shardingsphere/elasticjob/cloud/console/controller/search/JobEventRdbSearchTest.java]: Suggest a `final` class.

File: elasticjob-cloud/elasticjob-cloud-scheduler/src/test/java/org/apache/shardingsphere/elasticjob/cloud/console/controller/search/JobEventRdbSearchTest.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    
+    @Mock
+    private Connection conn;
+    
+    private JobEventRdbSearch.Condition condition;
+    
+    private JobEventRdbSearch jobEventRdbSearch;
+    
+    @Before
+    public void setUpClass() throws Exception {
```
Comment (dongzl) [inline elasticjob-cloud/elasticjob-cloud-scheduler/src/test/java/org/apache/shardingsphere/elasticjob/cloud/console/controller/search/JobEventRdbSearchTest.java]: Suggest the method name is `setUp`.

File: elasticjob-cloud/elasticjob-cloud-scheduler/src/test/java/org/apache/shardingsphere/elasticjob/cloud/console/controller/search/JobEventRdbSearchTest.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        fields.put("job_name", "TestJobName");
+        condition = new JobEventRdbSearch.Condition(0, 0, "job_name", "DESC", new Date(), new Date(), fields);
+        JobEventRdbSearch.Result<JobStatusTraceEvent> jobStatusTraceEvents = jobEventRdbSearch.findJobStatusTraceEvents(condition);
+        assertThat(jobStatusTraceEvents.getTotal(), is(1));
+        assertThat(jobStatusTraceEvents.getRows().size(), is(1));
+        assertThat(jobStatusTraceEvents.getRows().get(0).getJobName(), is("TestJobName"));
+        assertThat(jobStatusTraceEvents.getRows().get(0).getSource(), is(JobStatusTraceEvent.Source.LITE_EXECUTOR));
+        assertThat(jobStatusTraceEvents.getRows().get(0).getState(), is(JobStatusTraceEvent.State.TASK_RUNNING));
+    }
+}
```
Comment (dongzl) [inline elasticjob-cloud/elasticjob-cloud-scheduler/src/test/java/org/apache/shardingsphere/elasticjob/cloud/console/controller/search/JobEventRdbSearchTest.java]: Please add a blank line at end of file.

**Review by dongzl — CHANGES_REQUESTED** [review]

**Comment by Technoboy-** [comment]
Thanks for reviewing @dongzl

File: elasticjob-cloud/elasticjob-cloud-scheduler/src/test/java/org/apache/shardingsphere/elasticjob/cloud/console/controller/search/JobEventRdbSearchTest.java [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        fields.put("job_name", "TestJobName");
+        condition = new JobEventRdbSearch.Condition(0, 0, "job_name", "DESC", new Date(), new Date(), fields);
+        JobEventRdbSearch.Result<JobStatusTraceEvent> jobStatusTraceEvents = jobEventRdbSearch.findJobStatusTraceEvents(condition);
+        assertThat(jobStatusTraceEvents.getTotal(), is(1));
+        assertThat(jobStatusTraceEvents.getRows().size(), is(1));
+        assertThat(jobStatusTraceEvents.getRows().get(0).getJobName(), is("TestJobName"));
+        assertThat(jobStatusTraceEvents.getRows().get(0).getSource(), is(JobStatusTraceEvent.Source.LITE_EXECUTOR));
+        assertThat(jobStatusTraceEvents.getRows().get(0).getState(), is(JobStatusTraceEvent.State.TASK_RUNNING));
+    }
+
```
Comment (dongzl) [inline elasticjob-cloud/elasticjob-cloud-scheduler/src/test/java/org/apache/shardingsphere/elasticjob/cloud/console/controller/search/JobEventRdbSearchTest.java]: Please remove this blank line.

**Review by dongzl — CHANGES_REQUESTED** [review]

**Review by dongzl — APPROVED** [review]

**Review by terrymanu — APPROVED** [review]

**Pull request merged**